client/my-sites/plugins/education-footer/index.tsx

```diff
@@ -47,32 +49,66 @@ const EducationFooterContainer = styled.div`
 	}
 `;
 
+const MarketplaceContainer = styled.div< { isloggedIn: boolean } >`
+	--color-accent: #117ac9;
+	--color-accent-60: #0e64a5;
+
+	.marketplace-cta {
+		min-width: 122px;
+		margin-bottom: 26px;
+
+		@media ( max-width: 660px ) {
+			margin-left: 16px;
+			margin-right: 16px;
+		}
+	}
+
+	${ ( { isloggedIn } ) =>
+		! isloggedIn &&
+		`${ SectionContainer } {
+		padding-bottom: 0;
```

Comment:
Less padding is needed when the [LOHP footer](https://github.com/Automattic/wp-calypso/pull/67706) will be shown beneath this footer.